packages/e2e-test-app/test/__snapshots__/XAMLTest.test.ts.snap

```diff
@@ -0,0 +1,780 @@
+                                  "Foreground": "#E4000000",
+                                  "HorizontalAlignment": "Stretch",
+                                  "Left": 0,
+                                  "Margin": "0,0,0,0",
+                                  "Top": 0,
+                                  "VerticalAlignment": "Stretch",
+                                  "Visibility": "Visible",
+                                  "XamlType": "Windows.UI.Xaml.Controls.SymbolIcon",
+                                  "children": Array [
+                                    Object {
+                                      "Background": "#00000000",
+                                      "BorderBrush": null,
+                                      "BorderThickness": "0,0,0,0",
+                                      "Clip": null,
+                                      "CornerRadius": "0,0,0,0",
+                                      "FlowDirection": "LeftToRight",
+                                      "HorizontalAlignment": "Stretch",
+                                      "Left": 0,
+                                      "Margin": "0,0,0,0",
+                                      "Padding": "0,0,0,0",
+                                      "Top": 0,
+                                      "VerticalAlignment": "Stretch",
+                                      "Visibility": "Visible",
+                                      "XamlType": "Windows.UI.Xaml.Controls.Grid",
+                                      "children": Array [
+                                        Object {
+                                          "Clip": null,
+                                          "FlowDirection": "LeftToRight",
+                                          "Foreground": "#E4000000",
+                                          "HorizontalAlignment": "Stretch",
+                                          "Left": 0,
+                                          "Margin": "0,0,0,0",
+                                          "Padding": "0,0,0,0",
+                                          "Text": "",
+                                          "Top": 0,
+                                          "VerticalAlignment": "Center",
+                                          "Visibility": "Visible",
+                                          "XamlType": "Windows.UI.Xaml.Controls.TextBlock",
+                                        },
+                                      ],
+                                    },
+                                  ],
+                                },
+                              ],
+                            },
+                          ],
+                        },
+                      ],
+                    },
+                  ],
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    },
+  ],
+}
+`;
+
+exports[`ReactNativeXAML-MenuFlyout`] = `
+Object {
+}
+`;
```

Comment:
I think this bit is causing jest to fail

Reply:
yeah looks like snapshot names must end with a number (!!)